File: src/main/kotlin/SW01/HelloWorld.java
package SW01;

public class HelloWorld {

    private static final int a = 0;
    private static final int b = 1;

    public static void main(String[] args) {
        System.out.println("Hello World");

        int[] numbers = {a, b};

        for(int number : numbers) {
            System.out.println(number);
        }
    }

}

File: src/main/kotlin/SW01/HelloWorld.kt
package SW01

fun main() {

    println("Hello World")
    var number = 0
    number++
    println(number)

    val numbers = arrayOf(2, 0, 1)
    numbers.sort();
    for(number in numbers) {
        println(number)
    }

    fun sum(a: Int, b: Int) = a + b

    println(sum(2, 4))
}